src/intTest/resources/success_uat_data/NHS002/full_acceptance_without_nhsack.edifact.dat

```diff
@@ -0,0 +1,28 @@
+UNA:+.? '
+UNB+UNOB:2+110000044300001:80+110002063600005:80+030304:2347+428090++MEDRPT'
+UNH+000001+MEDRPT:0:1:RT:NHS002'
+BGM+LSR'
+DTM+137:200303042347:203'
+S01+01'
+NAD+SLA+++TRELISKE HAE/CH'
+SPR+DPT'
+S01+01'
+NAD+SLA+REF00:902'
+SPR+ORG'
+S02+02'
+GIS+N'
+RFF+SRI:428090/1'
+STS++UN'
+DTM+ISR:200303041714:203'
+S06+06'
+RFF+SSI:C361380'
+ADR++US:THE WINDMILL:WINDMILL HILL++PL1 1AA'
+S07+07'
+PNA+PAT+7000000007:OPI+++SU:MILLER+FO:WINDY+MI:BRIAN+TI:MR'
+DTM+329:19460918:102'
+PDI+1'
+S10+10'
+CIN+UN'
+FTX+CID+++MICROANEURYSM R EYE'
```

Comment:
Removed a number of segments temporarily until required parsers have been updated